inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -714,23 +678,85 @@ class ReadAndCompareAttributes : public ngraph::AttributeVisitor {
         }
     }
 
+    void verify_mem_buf(const std::string& name, const std::shared_ptr<ngraph::runtime::AlignedBuffer>& buffer) {
+        if (should_return()) {
+            return;
+        }
+        m_visited_attributes.insert(name);
+        const auto ref_value = m_attr_ref.get<storage::MemoryChunk>(name);
+        if (!ref_value) {
+            m_cmp_result += "missing attribute name: '" + name + "'";
+            return;
+        }
+
+        if (buffer->size() != ref_value->size() ||
+            std::memcmp(ref_value->data(), buffer->get_ptr(), ref_value->size()) != 0) {
+            m_cmp_result += "mismatch in value: '" + name + "' : look in to the mem buffer";
+            return;
+        }
+    }
+    using FunctionAccessor = ngraph::ValueAccessor<std::shared_ptr<ngraph::Function>>;
+
+    void verify_function(const std::string& name, FunctionAccessor& adapter) {
+        if (should_return()) {
+            return;
+        }
+        m_visited_attributes.insert(name);
+        const auto ref_value = m_attr_ref.get<std::shared_ptr<ngraph::Function>>(name);
+        if (!ref_value) {
+            m_cmp_result += "missing attribute name: '" + name + "'";
+            return;
+        }
+        Comparator c(m_check_flags);  // TODO check if m_check_flags might be move to equal::Equal
+        // TODO enable subgraph search in Comparator
+        const auto result = c.compare(*ref_value, adapter.get());
+        if (!result.valid) {
+            m_cmp_result += result.message;
+        }
+    }
+
+    void verify_else(const std::string& name, ngraph::ValueAccessor<void>& adapter) {
+        if (auto inputs =
+                ngraph::as_type<ngraph::AttributeAdapter<SubGraphOpInputDescription>>(&adapter)) {
+            verify(name, inputs->get());
+        } else if (
+            auto outputs =
+                ngraph::as_type<ngraph::AttributeAdapter<SubGraphOpOutputDescription>>(&adapter)) {
+            verify(name, outputs->get());
+        } else if (ngraph::is_type<ngraph::AttributeAdapter<SpecialBodyPorts>>(&adapter)) {
+            // drop comparison, no more info than port indexes which will be check in
+            // subgraph::compare_io
+        } else if (
+            auto a = ngraph::as_type<
+                ngraph::AttributeAdapter<std::shared_ptr<ngraph::runtime::AlignedBuffer>>>(
+                &adapter)) {
+            verify_mem_buf(name, a->get());
+        } else {
+            m_cmp_result += "compare attr [ ERR ]: " + name +
+                            " [drop `void` comparison which is '" + adapter.get_type_info().name +
+                            "']";
+        }
+    }
+    //-- DATA --
     const ReadAndStoreAttributes& m_attr_ref;
     Result m_cmp_result;
+    Comparator::CmpValues m_check_flags;
     std::set<AttrName> m_visited_attributes;
-    bool m_fast_exit{true};
+    static const bool m_fast_exit{true};
```

Comment:
Why did you make this member to static?

Reply:
It may be `constexpr` but this is not c++17. This class is only in single compilation unit and `static` won't affect others (I'm not expect any linking issues for integral types).
Before c++17 you use both aproach in templates:
```
enum { value = 1 };
```
```
static const bool value = true;
```

@ilyachur , should I revert this change?